server/datastore/mysql/apple_mdm.go

```diff
@@ -134,7 +136,15 @@ GROUP BY
 HAVING
 	COUNT(*) = ?`
```

Comment:
@roperzh I think that if , on the contrary, we want to match only teams that _do_ have both of the Fleet-specific profiles, then we'd still have the NOT EXISTS as-is, `p1.identifier NOT IN (?)` removed, the HAVING COUNT(*) would be `len(hexMD5Hashes) + len(fleetIdents)` and an additional condition `(SELECT COUNT(*) FROM mdm_apple_configuration_profiles p3 WHERE p3.team_id = p1.team_id AND p3.identifier IN (...fleetIdents...)) = <len(fleetIdents)>` (untested, but you get the idea).